src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay/configuration/setup-session-replay.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: "Setup session replay"
+metaDescription: "Enable session replay to play back user interactions and improve page performance."
+freshnessValidatedDate: 2024-12-19
+---
+
+Fully utilize session replay in New Relic, start by enabling this feature in your browser application settings. This step is essential for capturing user interactions on your web app, providing insights into user journeys, and troubleshooting issues effectively. Follow the steps below to enable session replay and ensure your browser agent is configured correctly.
+
+## Enable session replay [#enable-session-replay]
+
+    1. Go to <DNT>**[one.newrelic.com](https://one.newrelic.com/all-capabilities) > All Capabilities > Browser**</DNT>.
+
+    2. Select your browser app.
+
+    3. In the left-hand menu, click <DNT>**Application settings**</DNT>.
+    
+    4. On the <DNT>**Application settings**</DNT> page, make sure <DNT>**Pro**</DNT> or <DNT>**Pro + SPA**</DNT> browser agent is selected. Session replay is not available for the Lite browser agent.
+
+    5. Toggle ON <DNT>**Session tracking**</DNT> and <DNT>**Session replay**</DNT> settings.
+
+## Configure sampling rates [#configure-sampling-rates]
```

Comment:
This section title starts with a verb but doesnt have any actions within. Revisit.

Reply:
Updated